client/signup/controller.js

```diff
@@ -97,16 +97,19 @@ export const removeP2SignupClassName = function () {
 
 export default {
 	redirectTests( context, next ) {
+		const currentFlowName = getFlowName( context.params );
```

Comment:
```suggestion
		const currentFlowName = getFlowName( context.params );
		currentFlowName === 'onboarding' && loadExperimentAssignment( 'refined_reskin_v1' );
```

If you do use the `isEligible` you would need to move the prefetching here to make it conditional on the flow.